io.github.maddecoder.Classic-RBDOOM-3-BFG.yml

```diff
@@ -0,0 +1,139 @@
+app-id: io.github.maddecoder.Classic-RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "22.08"
```

Comment:
current runtime is 23.08

Reply:
FFMPEG seems to have an issue with 23.08

Reply:
> FFMPEG seems to have an issue with 23.08

What issue? If build faliure is related to mathops.h, use this [patch](https://github.com/flathub/io.github.celluloid_player.Celluloid/blob/master/ffmpeg-binutils.patch)

Reply:
Got the patch and it compiled. Also why bbhtt's messages are not visible on GIthub?